Week_6/sentimental-readability/readability.py

```diff
@@ -0,0 +1,41 @@
+import cs50
+
+def main():
+    while True:
+        str = cs50.get_string('Text: ')
+        index = 0
+        i = 0
+        alphabets = 0
+        digits = 0
+        special = 0
+        words = 1
+        sentences = 0
+        
+        if (str):
+            for i in range(len(str)):
+                if (str[i].isalpha()):
+                    alphabets += 1
+                elif (str[i].isnumeric()):
+                    digits += 1
+                elif (str[i] == ' ' and str[i + 1]):
+                    words += 1
+                elif (str[i] == '?' or str[i] == '!' or str[i] == '.'):
+                    sentences += 1
+                else:
+                    special += 1
+
+            L = float(alphabets / words * 100)
+
+            S = float(sentences / words * 100)
```

Comment:
Try to think of better variable names :)

Reply:
It was a requirement in the task to use this :)